srcs/server/prisma/migrations/20220827134504_/migration.sql

```diff
@@ -0,0 +1,18 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `two_fa_activated` on the `Credentials` table. All the data in the column will be lost.
+  - You are about to drop the column `two_fa_secret` on the `Credentials` table. All the data in the column will be lost.
+  - You are about to drop the column `two_fa_activated` on the `User` table. All the data in the column will be lost.
+  - You are about to drop the column `two_fa_secret` on the `User` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "Credentials" DROP COLUMN "two_fa_activated",
+DROP COLUMN "two_fa_secret",
+ADD COLUMN     "twoFactorActivated" BOOLEAN NOT NULL DEFAULT false,
+ADD COLUMN     "twoFactorSecret" TEXT;
+
+-- AlterTable
+ALTER TABLE "User" DROP COLUMN "two_fa_activated",
+DROP COLUMN "two_fa_secret";
```

Comment:
You can name the migrations using the command :
`npx prisma migrate dev --name 2fa_on_credentials`

for example. It is usefull as we will have a lot of migrations at the end of the project to quicky identify them.

Reply:
I think it's a better idea to name them as well!

But the migration was done automatically when I launched the makefile.

Next time I'll make sure to migrate --name before relaunching a docker compose.